libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts

```diff
@@ -28,6 +28,7 @@ export interface DotMenuItem extends Omit<MenuItem, 'icon'> {
     icon: string | SafeUrl;
     isActive?: () => boolean;
     attributes?: Record<string, unknown>;
+    data?: Record<string, unknown>;
```

Comment:
Why `Record<string, unknown>;`? it will not be a just `contentlet`? Or does it receive more data?